packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors/ColorSwatch.css

```diff
@@ -1,17 +1,17 @@
 .ws-color-swatch {
```

Comment:
We may want to review this stylesheet and see if there is anything we can do differently. Though with these existing styles, we'll want to use `--pf-t` tokens instead of `pf-v6-global` vars, and potentially a few other tweaks - tokens for border-radius, font-weight, potentially the 44px flex-basis on the `<svg>` converted to an icon size token.